cmd/oras/internal/display/status/interface.go

```diff
@@ -61,3 +63,24 @@ type CopyHandler interface {
 	PostCopy(ctx context.Context, desc ocispec.Descriptor) error
 	OnMounted(ctx context.Context, desc ocispec.Descriptor) error
 }
+
+// ManifestIndexCreateHandler handles status output for manifest index create command.
+type ManifestIndexCreateHandler interface {
+	OnSourceManifestFetching(source string) error
+	OnSourceManifestFetched(source string) error
+	OnIndexPacked(shortDigest string) error
```

Comment:
Packed index should be moved to metadata output.

Reply:
Packed, Updated, Pushed, Added, Removed, Merged should all be moved to metadata output.

Reply:
Use one interface for packed and updated.

Reply:
Created `manifestPackHandler` interface that contains `OnIndexPacked`, `OnIndexPushed` and `OnCompleted`. Removed `OnIndexUpdate` and replaced it with `OnIndexPacked`.